cpp/src/Slice/Grammar.y

```diff
@@ -1706,11 +1710,6 @@ enumerator
     auto ens = make_shared<EnumeratorListTok>(); // Dummy
     $$ = ens;
 }
-| %empty
-{
-    auto ens = make_shared<EnumeratorListTok>();
-    $$ = ens; // Dummy
-}
```

Comment:
The culprit for the comma parsing bug.
It allowed 'empty' enumerators to appear between commas: `Red,,,, White, Blue`.

Now we allow the entire `enumerator_list` to be empty.
Much more sensible.